examples/angular/auto-refetching/package.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "@tanstack/query-example-angular-auto-refetching",
+  "type": "module",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "watch": "ng build --watch --configuration development"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/common": "^17.3.12",
+    "@angular/compiler": "^17.3.12",
+    "@angular/core": "^17.3.12",
+    "@angular/platform-browser": "^17.3.12",
+    "@angular/platform-browser-dynamic": "^17.3.12",
+    "@tanstack/angular-query-experimental": "^5.61.1",
+    "rxjs": "^7.8.1",
+    "tslib": "^2.6.3",
+    "zone.js": "^0.14.8"
+  },
+  "devDependencies": {
+    "@angular-devkit/build-angular": "^17.3.8",
+    "@angular/cli": "^17.3.8",
+    "@angular/compiler-cli": "^17.3.12",
+    "typescript": "5.3.3"
+  }
```

I updated the other examples to Angular 19.1 pre-release and TS 5.7

```suggestion
  "dependencies": {
    "@angular/common": "^19.1.0-next.0",
    "@angular/compiler": "^19.1.0-next.0",
    "@angular/core": "^19.1.0-next.0",
    "@angular/platform-browser": "^19.1.0-next.0",
    "@angular/platform-browser-dynamic": "^19.1.0-next.0",
    "@tanstack/angular-query-experimental": "^5.62.0",
    "rxjs": "^7.8.1",
    "tslib": "^2.6.3",
    "zone.js": "^0.15.0"
  },
  "devDependencies": {
    "@angular/build": "^19.0.2",
    "@angular/cli": "^19.0.2",
    "@angular/compiler-cli": "^19.1.0-next.0",
    "typescript": "5.7.2"
  }
```